test/fluree/db/json_ld/credential_test.cljc

```diff
@@ -0,0 +1,127 @@
+(ns fluree.db.json-ld.credential-test
```

Comment:
These tests hew a bit close towards testing the implementation, but for now they document how to use the api.